veps/sig-compute/hypervisor-abstraction.md

```diff
@@ -0,0 +1,577 @@
+# VEP #97: Hypervisor Abstraction Layer
```

Comment:
nit: I think that using the word "accelerator" instead of "hypervisor" would be clearer as it would emphasize we don't aim to replace libvirt or QEMU at this point, only KVM.

Reply:
In old oVirt days, even the manager that integrated with libvirt was called a hypervisor.
From my perspective, the virt-launcher pod is a hypervisor. This VEP discusses a way to support alternative hypervisors, which can cover the virt-launcher as a whole or the way it is composed.